pynsee/download/_download_store_file.py

```diff
@@ -28,11 +33,27 @@ def _download_store_file(tempdir: tempfile.TemporaryDirectory, id: str):
     """
 
     dict_data_source = _get_dict_data_source()
+    caract = None
     if id in dict_data_source.keys():
         caract = dict_data_source[id]
     else:
-        suggestions = difflib.get_close_matches(id, dict_data_source.keys())
 
+        if id.rsplit("_", 1)[-1].lower() == "latest":
```

Comment:
switch to elif

Reply:
maybe adding an explanatory comment would be good, I understand it from the PR topic, but out of context it might have taken a while